src/ResourceManagement/ApiManagement/ApiManagementManagement/Microsoft.Azure.Management.ApiManagement.nuget.proj

```diff
@@ -5,7 +5,7 @@
     Microsoft.Azure.Management.ApiManagement
     -->
     <SdkNuGetPackage Include="Microsoft.Azure.Management.ApiManagement">
-      <PackageVersion>3.0.0-preview</PackageVersion>
+      <PackageVersion>3.0.1-preview</PackageVersion>
```

Comment:
Updated the nuget to 3.1.0 and added validation on the entities. We cannot create a test, as we do not allow set on the two properties. Only services with existing setup can view the properties (subnetname and VnetId).